lib/rules/value-list-comma-newline-after/index.js

```diff
@@ -60,7 +60,7 @@ function rule(expectation, options, context) {
 				  }
 				: null,
 			determineIndex: (declString, match) => {
-				const nextChars = declString.substr(match.endIndex, declString.length - match.endIndex);
+				const nextChars = declString.substring(match.endIndex, declString.length);
```

Comment:
[note] [`substr()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr) is deprecated.